pkg/kubelet/preemption/preemption.go

```diff
@@ -191,8 +191,9 @@ func (a admissionRequirementList) distance(pod *v1.Pod) float64 {
 		remainingRequest := float64(req.quantity - resource.GetResourceRequest(pod, req.resourceName))
 		if remainingRequest < 0 {
 			remainingRequest = 0
+		} else {
```

Comment:
Could we get rid of the `if/else` and instead write this as:

```
if remainingRequest > 0 {
    dist += math.Pow(remainingRequest/float64(req.quantity), 2)
}
```

Reply:
Changed accordingly.